src/scripts/pbs_status.py

```diff
@@ -226,14 +229,11 @@ def qstat(jobid=""):
     Returns a python dictionary with the job info.
     """
     qstat_bin = get_qstat_location()
-    command = (qstat_bin, '--version')
-    qstat_process = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.STDOUT)
-    qstat_version, _ = qstat_process.communicate()
 
     starttime = time.time()
     log("Starting qstat.")
     command = (qstat_bin, '-f')
-    if not re.search(r'PBSPro', qstat_version):
+    if os.environ.get('pbs_pro') != 'yes':
```

Comment:
Just for sanity, could you set to set the string to "lower" before being compared.